burstphoto/texture/texture.metal

```diff
@@ -546,14 +572,38 @@ kernel void normalize_texture(texture2d<float, access::read_write> in_texture [[
     in_texture.write(in_texture.read(gid).r/(norm_texture.read(gid).r + norm_scalar), gid);
 }
 
+/**
+ Used for calculating texture_mean. Can't use the UInt version since the images at this point are stored as floats.
+ */
+kernel void sum_rect_columns_float(texture2d<float, access::read> in_texture [[texture(0)]],
+                                   texture2d<float, access::write> out_texture [[texture(1)]],
+                                   constant int& top [[buffer(1)]],
```

Comment:
I fixed the indices, good catch.

However I don't think that changing the names, and changing to signature to take `height` instead of `bottom` to fit the other usage would be an improvement. This loop becomes messier to write, (the other pad_top, pad_left calls don't have to use loops.)